manager/src/main/java/org/openremote/manager/rules/JsonRulesBuilder.java

```diff
@@ -785,6 +788,7 @@ protected RuleActionExecution buildRuleActionExecution(JsonRule rule, RuleAction
 
             Notification notification = ValueUtil.clone(notificationAction.notification);
             String body;
+            PushNotificationAction action;
```

Comment:
I think you can move this variable declaration down, to a spot after `if (isPush) { }`,
as this variable isn't use elsewhere.